course/grades.py

```diff
@@ -295,43 +309,207 @@ def grade_key(entry):
         })
 
 
+class ExportGradeBookForm(StyledForm):
+    def __init__(self, *args, **kwargs):
+        # type: (*Any, **Any) -> None  # noqa
+        super(ExportGradeBookForm, self).__init__(*args, **kwargs)
+
+        self.fields["user_info_fields"] = forms.ChoiceField(
+            choices=(csv_settings.export_csv_fields_options),
+            initial=csv_settings.export_csv_fields_options[0],
+            label=_("User Attributes to Export"),
+        )
+
+        self.fields["exclude_course_staff"] = forms.BooleanField(
+            required=False,
+            initial=True,
+            label=_("Exclude Course Staff"),
+            help_text=_(
+                "If selected, grades of course staff (participations who "
+                "have '%(permission)s' permissions) will be excluded."
+                % {"permission": pperm.assign_grade}),
+        )
+
+        self.fields["zero_for_state_none"] = forms.BooleanField(
+            required=False,
+            initial=False,
+            label=_("0 percentage for state 'NONE'"),
+            help_text=string_concat(_(
+                "Use 0 percentage for opportunities with grade state `NONE`, "
+                "'NONE' here means there're no grades generated for the "
+                "opportunity of that user."),
+                " ",
+                _("Note: this won't be affected by \"Maximum percentage value\" "
+                  "and \"Minimum percentage value\" below."))
+        )
+
+        self.fields["zero_for_graded_none"] = forms.BooleanField(
+            required=False,
+            initial=True,
+            label=_("0 percentage for graded 'NONE'"),
+            help_text=string_concat(
+                _("Use 0 percentage for graded opportunities which have "
+                  "`NONE` points."),
```

Comment:
"`NONE` points" reads awkwardly. Maybe "that do not currently have a defined point count."

Reply:
That's fair. I think then we should notice use if `False`, it will result in `NONE` in the csv.